docs/debugging.rst

```diff
@@ -60,20 +60,11 @@ When an application is built, ``flatpak-builder`` automatically
 creates a ``.Debug`` extension. This can be disabled with the ``no-debuginfo``
 option.
 
-To install the ``.Debug`` extension created locally, export the build
-to a repo by passing ``--repo repo`` to ``flatpak-builder`` when
-building. Then add the repo folder as a local remote::
+To install the ``.Debug`` extension created locally, pass ``--install``
+to ``flatpak-builder`` which will set up a new remote ``foo-origin``
```

Comment:
it might be a different name to be unique. This can be checked with `flatpak remotes --columns=name,url`